src/libstd/sync/mpsc/mod.rs

```diff
@@ -487,18 +494,23 @@ pub fn channel<T>() -> (Sender<T>, Receiver<T>) {
 
 /// Creates a new synchronous, bounded channel.
 ///
-/// Like asynchronous channels, the `Receiver` will block until a message
+/// Like asynchronous channels, the [`Receiver`] will block until a message
 /// becomes available. These channels differ greatly in the semantics of the
 /// sender from asynchronous channels, however.
 ///
 /// This channel has an internal buffer on which messages will be queued. `bound`
 /// specifies the buffer size. When the internal buffer becomes full, future sends
 /// will *block* waiting for the buffer to open up. Note that a buffer size of 0
-/// is valid, in which case this becomes  "rendezvous channel" where each send will
-/// not return until a recv is paired with it.
+/// is valid, in which case this becomes  "rendezvous channel" where each [`send`]
```

Comment:
Being french myself I had my doubts about that but in English it's definitely rendezvous : 

https://fr.wikipedia.org/wiki/Rendez-vous
https://en.wikipedia.org/wiki/Rendezvous

For some unknown and I guess historical reasons the translators dropped the dash